assets/js/components/ClustersList.jsx

```diff
@@ -71,7 +95,23 @@ function ClustersList() {
         filterFromParams: true,
         filter: (filter, key) => (element) =>
           element[key].some((sid) => filter.includes(sid)),
-        render: (_, { sid }) => sid.join(', '),
+        render: (_, { sid }) =>
+          sid.map((singleSid) => {
+            const linkData = getSapSystemLinkBySID(
+              applicationInstances,
+              databaseInstances,
+              singleSid
+            );
+
+            return (
+              <SapSystemLink
```

Comment:
`map`ping and returning JSX elements should require a key prop to be passed, can you do that?